update autoscaling (#105)
huggingface/hf-endpoints-documentation: docs/source/autoscaling.mdx

### Scaling based on pending requests (beta feature)

> You can change the scaling criteria to be based on pending requests instead of utilization metrics. This is currently an experimental feature and we advice caution if you're using it for production workloads.

Reviewer: ```suggestion
You can change the scaling criteria to be based on pending requests instead of utilization metrics. This is currently an experimental feature and we advise caution if you're using it for production workloads.
```
or how about: and we advise testing prior to using it for production workloads.

Author: "and we advise testing prior to using it for production workloads"

I like it, it's even less "worry-triggering" 👍

- pending requests are requests that have not yet received an HTTP status, meaning they include in-flight requests and requests currently being processed.
- by default, if there are more than 1.5 pending requests per replica in the past 20 seconds, it triggers an autoscaling event and adds a replica to your deployment.

>     - you can adjust this threshold to meet your specific requirements from the settings page.

Reviewer: ```suggestion
    - you can adjust this threshold to meet your specific requirements under Endpoint settings.
```
